docs/index.rst

```diff
@@ -22,7 +22,7 @@ Understand messages, hold conversations, and connect to messaging channels and A
    :hidden:
 
    installation
-   project-structure
```

Comment:
I think this the right place to have the tutorial, but I think it would be good to add a link to installations page which would lead to this tutorial